aiosmtpd/smtp.py

```diff
@@ -73,12 +74,6 @@ def __init__(self, handler,
         self.event_handler = handler
         self.data_size_limit = data_size_limit
         self.enable_SMTPUTF8 = enable_SMTPUTF8
-        if enable_SMTPUTF8:
-            if decode_data:
-                raise ValueError(
-                    "decode_data and enable_SMTPUTF8 cannot be set to "
-                    "True at the same time")
-            decode_data = False
```

Comment:
They no conflict now. If you will enable decode data and disable SMTPUTF8 it will decode in strict-ascii mode.